src/TorchSharp/NN/Activation/Softsign.cs

```diff
@@ -57,12 +47,11 @@ public static partial class functional
                 /// Softsign
                 /// </summary>
                 /// <param name="x">The input tensor</param>
-                /// <returns></returns>
-                public static Tensor Softsign(Tensor x)
+                /// <param name="inplace">Do the operation in-place. Default: False</param>
+                public static Tensor softsign(Tensor x, bool inplace = false)
                 {
-                    using (var m = nn.Softsign()) {
-                        return m.call(x);
-                    }
+                    using var y = 1 + x.abs();
+                    return inplace ? x.div_(y) : x.div(y);
```

Comment:
The x.abs() isn't being disposed, since the `using` is only placed on (1 + x.abs()), which is a new tensor. 
Also, missing `.alias()`